tests/integration/tls_tests/helpers.py

```diff
@@ -122,35 +128,26 @@ def process_ls_time(ls_output):
     time_as_str = "T".join(ls_output.split("\n")[0].split(" ")[5:7])
     # further strip down additional milliseconds
     time_as_str = time_as_str[0:-3]
-    d = datetime.strptime(time_as_str, "%Y-%m-%dT%H:%M:%S.%f")
-    return d
+    return datetime.strptime(time_as_str, "%Y-%m-%dT%H:%M:%S.%f")
 
 
-def process_systemctl_time(systemctl_output):
+def process_systemctl_time(systemctl_output) -> datetime:
     """Parse time representation as returned by the 'systemctl' command."""
     "ActiveEnterTimestamp=Thu 2022-09-22 10:00:00 UTC"
     time_as_str = "T".join(systemctl_output.split("=")[1].split(" ")[1:3])
-    d = datetime.strptime(time_as_str, "%Y-%m-%dT%H:%M:%S")
-    return d
+    return datetime.strptime(time_as_str, "%Y-%m-%dT%H:%M:%S")
 
 
-async def scp_file_preserve_ctime(ops_test: OpsTest, unit_name: str, path: str) -> int:
-    """Returns the unix timestamp of when a file was created on a specified unit."""
+async def scp_file_preserve_ctime(ops_test: OpsTest, unit_name: str, path: str) -> str:
```

Comment:
I could be wrong - but it looks like this function is actually just used to read a file via copying over the contents. Instead I think we could remove it and adopt my suggestions below

Reply:
You're right! I'll rename the helper to just `scp_file` and keep it there in case we need in in the future, but I'll switch to your suggestion